cmd/ros-installer/main.go

```diff
@@ -14,7 +14,7 @@ import (
 var (
 	automatic   = flag.Bool("automatic", false, "Check for and run automatic installation")
 	printConfig = flag.Bool("print-config", false, "Print effective configuration and exit")
-	configFile  = flag.String("config-file", "", "Config file to use, local file or http/tftp URL")
+	configFile  = flag.String("config-file", "/oem/userdata", "Config file to use, local file or http/tftp URL")
```

Comment:
I am curious about that. What's the content of `/oem/userdata`? If this is created by the datasource yip plugin and then loaded again as a yip config file I think it should be  `/oem/userdata.yaml`.

Reply:
this is expected to be the user input only without any further processing. I've checked the code and `userdata.yaml` and `userdata` are equivalent (there is no conversion happening, while I was expecting it on the first place too)

Reply:
There is no conversion. The only difference is that  `userdata.yaml` is granted to be readable by yip (extended or not). But probably this doesn't make a difference here.